config/routes.rb

```diff
@@ -320,6 +318,8 @@
   #########
 
   get '/tools/bulk/relationships' => "tools#bulk_relationships"
+  get '/tools/merge' => "tools#merge_entities"
+  post '/tools/merge' => "tools#merge_entities!"
```

Comment:
see comment above. the very existence of a `tools` endpoint feels like a code smell to me. but i'm not inclined to be a grumpus about it. ;)